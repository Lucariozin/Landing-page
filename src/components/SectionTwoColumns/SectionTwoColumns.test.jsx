import { render, screen } from '@testing-library/react';
import { SectionTwoColumns } from '.';
import Mock from './mock';

describe('<SectionTwoColumns />', () => {
  it('should render <SectionTwoColumns /> correctly', () => {
    render(<SectionTwoColumns {...Mock} />);

    expect(screen.getByText('January brings us firefox 85'));
    expect(screen.getByText(/CSS and associated devtools/i));
    expect(screen.getAllByAltText('Uma imagem qualquer'));
  });

  it('should to match snapshot', () => {
    const { container } = render(<SectionTwoColumns {...Mock} />);
    expect(container).toMatchSnapshot();
  });
});
