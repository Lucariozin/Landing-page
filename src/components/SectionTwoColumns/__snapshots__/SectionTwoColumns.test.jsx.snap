// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTwoColumns /> should to match snapshot 1`] = `
.c2 {
  display: grid;
  grid-template-columns: 1.3fr 2fr;
  gap: 2rem;
  margin-top: 3rem;
}

.c3 {
  margin-bottom: 5rem;
}

.c6 {
  width: 100%;
}

.c0 {
  width: 100%;
  min-height: 100vh;
  padding: 1rem;
  padding-top: 8rem;
  background-image: url(assets/img/background.svg);
  background-position: center center;
  background-size: cover;
  color: var(--white);
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
}

.c4 {
  color: var(--white);
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c5 {
  color: var(--white);
  font-style: normal;
  font-size: 2rem;
}

@media (max-width:800px) {
  .c2 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    text-align: center;
  }
}

@media (max-width:768px) {
  .c4 {
    font-size: 4rem;
  }
}

<div>
  <div
    class="c0"
    id=""
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class=""
        >
          <div
            class="c3"
          >
            <h2
              class="c4"
            >
              January brings us firefox 85
            </h2>
          </div>
          <p
            class="c5"
          >
            To wrap up January, we are proud to bring you the release of Firefox 85. In this version we are bringing you support for the :focus-visible pseudo-class in CSS and associated devtools, and the complete removal of Flash support from Firefox.
          </p>
        </div>
        <img
          alt="Uma imagem qualquer"
          class="c6"
          src="assets/img/javascript.svg"
        />
      </div>
    </div>
  </div>
</div>
`;
